### PR TITLE
Humble update to the FAQ section

### PR DESCRIPTION
**Pull request by smolinari**
And some smaller corrections.

Hope it's ok and keep up the great work! 😊 

Scott

### CODE REVIEW

File: active-rfcs/0000-function-api.md [hunk 1]
```diff
@@ -17,19 +17,19 @@ We are here to serve the Vue community and that is the purpose of an RFC - (Requ
...
 
 ## This will lead to spaghetti code and is much harder to read.
 
 Please read [this section](#spaghetti-code-in-inexperienced-hands).
 
 ## The Class API is much better!
 
-We've [taken several weeks and looked in depth at creating a Class API to only find that it doesn't profice what we hoped it would.](https://github.com/vuejs/rfcs/blob/function-apis/active-rfcs/0000-function-api.md#type-issues-with-class-api) [Evan also explained in detail about why the Class API was no longer the right direction.](https://github.com/vuejs/rfcs/pull/17#issuecomment-494242121).
+We've [taken several weeks and looked in depth at creating a Class API to only find that it doesn't profide what we hoped it would.](https://github.com/vuejs/rfcs/blob/function-apis/active-rfcs/0000-function-api.md#type-issues-with-class-api) [Evan also explained in detail about why the Class API was no longer the right direction.](https://github.com/vuejs/rfcs/pull/17#issuecomment-494242121).
```
Comment (yyx990803) [inline active-rfcs/0000-function-api.md]: provide?

Comment (smolinari) [reply active-rfcs/0000-function-api.md]: Yes. I thought I corrected that in a second commit. 

Scott

Comment (smolinari) [reply active-rfcs/0000-function-api.md]: Oh. I see I corrected it incorrectly. Sorry about that. 

Scott

**Comment by yyx990803** [comment]
There seems to be a number of conflicts as I just edited the same sections. Would you please take a look at the latest and maybe submit a separate PR if still necessary?

**Comment by smolinari** [comment]
If you think what I wrote could be ok to make the change, I'll do it. 

The only thing I'd correct for sure is unexperience in inexperience (in the spaghetti code bit). 

Scott

**Pull request closed without merging**